src/classes/GE_LookupController_TEST.cls

```diff
@@ -92,21 +92,26 @@ private class GE_LookupController_TEST {
 
     @IsTest
     static void testSearchSOQL() {
-        List<Account> accounts = new List<Account>{
-                new Account(Name = 'Test Account'),
-                new Account(Name = 'Non-Matching Account')
+        TEST_OpportunityBuilder builder = new TEST_OpportunityBuilder()
+            .withClosedWonStage()
+            .withCloseDate(Date.today());
+
+        List<Opportunity> opps = new List<Opportunity> {
+            builder.withName('Test Opportunity').build(),
+            builder.withName('Non-Matching Opportunity').build()
         };
-        insert accounts;
+        
+        insert opps;
 
         // doSearch utilizes SOSL, which means results must be hardcoded in tests
-        Test.setFixedSearchResults(new List<Id>{accounts[0].Id});
+        Test.setFixedSearchResults(new List<Id>{opps[0].Id});
 
         Test.startTest();
-        List<GE_LookupController.LookupResult> results = GE_LookupController.doSearchSOQL('test', 'Account');
+        List<GE_LookupController.LookupResult> results = GE_LookupController.doSearchSOQL('test', 'Opportunity');
```

Comment:
@patrick-yan-sf , by this change the the GE controller test class will pass on an encrypted org. Not sure if you have a related WI and if more changes are pending.

@andrewyu-salesforce , thanks for making changes.